### PR TITLE
Rename abrt tests

### PR DESCRIPTION
**Pull request by sthirugn**
Renamed as per issue #3083

### CODE REVIEW

File: tests/foreman/cli/test_abrt.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def test_abrt_1(self):
 
         """
 
-    def test_abrt_2(self):
+    def test_positive_create_multiple_reports(self):
```
Comment (omaciel) [inline tests/foreman/cli/test_abrt.py]: Does this create a ``crash`` report as well as `test_positive_create_crash_report`? If so, perhaps we want to rename this to `test_positive_create_reports`?

File: tests/foreman/cli/test_abrt.py [hunk 2]
```diff
@@ -76,7 +76,7 @@ def test_abrt_4(self):
 
         """
 
-    def test_abrt_5(self):
+    def test_positive_cli_report(self):
```
Comment (omaciel) [inline tests/foreman/cli/test_abrt.py]: What is the difference between this report generating method and the others? They're all using the CLI so this name doesn't really tell me what is going on. Should this be a `test_positive_search_report_by_id` or something similar?

File: tests/foreman/cli/test_abrt.py (hunk 1 above)
Comment (sthirugn) [inline tests/foreman/cli/test_abrt.py]: The test case does not have enough information about it.  But you made a right point.  I am going to call these tests `test_positive_create_report` and `test_positive_create_reports` now.

**Comment by sthirugn** [comment]
@omaciel pr updated

File: tests/foreman/cli/test_abrt.py (hunk 2 above)
Comment (sthirugn) [inline tests/foreman/cli/test_abrt.py]: To be honest, I have no idea about it since the test cases are not implemented and have very less info now.

Comment (sthirugn) [inline tests/foreman/cli/test_abrt.py]: Actually I can update with the name you suggested.

**Comment by omaciel** [comment]
ACK

**Pull request merged**